DevEdu.API/DevEdu.API/Controllers/TaskController.cs

```diff
@@ -219,5 +250,17 @@ public List<GroupTaskInfoWithGroupOutputModel> GetGroupsByTaskId(int taskId)
             var output = _mapper.Map<List<GroupTaskInfoWithGroupOutputModel>>(dto);
             return output;
         }
+
+        // api/task/group/user/answer-on-task-in-group}
+        [HttpGet("group/user/answer-on-task-in-group")]
+        [Description("Get all student task answers by groups")]
+        [ProducesResponseType(typeof(List<StudentAnswerOnlyAnswersOutputModel>), StatusCodes.Status200OK)]
+        public List<StudentAnswerOnlyAnswersOutputModel> GetAnswersForStudentInGroup(int userId)

```

Comment:
lets rename it to GetAllAnswersByStudentId and will return all answers without filter by group